Model/IndicesConfigurator.php

```diff
@@ -126,10 +83,10 @@ public function saveConfigurationToAlgolia(int $storeId, bool $useTmpIndex = fal
         $this->algoliaHelper->waitLastTask();
 
         $this->setProductsSettings($storeId, $useTmpIndex);
-        $this->algoliaHelper->waitLastTask();
```

Comment:
Indeed, the `waitLastTask() `calls are handled by the `ProductHelper`. If I let this one, the `saveConfigurationToAlgolia` call hangs forever in case we use Multi App IDs

Reply:
Good to know! Thanks for the added context. 🙁